update readme (#108)
actnforchildren/mental_health_app: README.md

> the Action for Children application allow users to record their emotions.

Reviewer: ```suggestion
The Action for Children application allow users to record their emotions.
```

![image](https://user-images.githubusercontent.com/6057298/48003870-9a077200-e107-11e8-92d0-9095e95b615f.png)

> The daily, weekly or monthly view show a resume of the logged emotions

Reviewer: ```suggestion
The daily, weekly and monthly views allow the user to see a record of their previously logged emotions.
```

- create the database
  `docker-compose run --rm app mix ecto.create`

- run the migrations:
  `docker-compose run --rm app mix ecto.migrate`

- create/add users:
  `docker-compose run --rm app mix run priv/repo/create_test_users.exs`

- run the application with
  `docker-compose up`

- visit the app on localhost:4000

## Deploy on Heroku

To create a new heroku app you can follow the Phoenix guide https://hexdocs.pm/phoenix/heroku.html

> Check that the heroku remote is defined `git remote -v`. For more information you can read the heroku deployement guide: https://devcenter.heroku.com/articles/git#creating-a-heroku-remote

Reviewer: One tiny typo 'deployement' should be > 'delpoyment'
```suggestion
Check that the heroku remote is defined `git remote -v`. For more information you can read the heroku deployment guide: https://devcenter.heroku.com/articles/git#creating-a-heroku-remote
```
